app/views/editor/verifier/VerifierTest.js

```diff
@@ -115,13 +114,33 @@ module.exports = (VerifierTest = class VerifierTest extends CocoClass {
   }
 
   setupGod () {
-    this.god.setLevel(this.level.serialize({ supermodel: this.supermodel, session: this.session, otherSession: null, headless: true, sessionless: false }))
+    const level = this.level.serialize({ supermodel: this.supermodel, session: this.session, otherSession: null, headless: true, sessionless: false })
+    const hero = this.world.getThangByID('Hero Placeholder')
+    if (this.checkClampedProperties) {
+      level.constrainHeroHealth = true
+      if (!this.clampedProperties.inited) {
+        this.clampedProperties = {
+          inited: true,
+          maxHealth: { current: 1, lower: 1, upper: hero.maxHealth },
+          maxSpeed: { current: 1, lower: 1, upper: hero.maxSpeed },
```

Comment:
Slowest possible speed is something like 3. We also need to check to see if the hero can have too much speed to beat the level, so upper could be like 25.

Reply:
ya, i would add upper bound checking later. i just want to first made a good logic to check lowest one by one, then copy that for upper bound